src/features/tags/components/TagManager/components/TagSelect/ValueTagForm.tsx

```diff
@@ -46,7 +46,10 @@ const ValueTagForm: React.FC<{
           <TagChip tag={{ ...tag, value: inputValue }} />
         </Box>
       </Box>
-      <ZUISubmitCancelButtons onCancel={onCancel} />
+      <ZUISubmitCancelButtons
+        onCancel={onCancel}
+        submitDisabled={inputValue === '' ? true : false}
```

Comment:
I would do this:

```suggestion
        submitDisabled={!Boolean(inputValue)}
```

Or perhaps this:

```suggestion
        submitDisabled={!inputValue}
```

Note that when we discussed this in the room, I mistook the `submitDisabled` for `submitEnabled` and used `!!` to convert a truthy (non-empty) string to `true` (the boolean). That was a mistake. When the string is truthy (non-empty) we want it to become `false`, i.e. NOT disabled.

The more important change here however is removing the ternary expression, which is not necessary because the condition (left of `?`) already evaluates to either `true` or `false` on it's own.

I also wanted to remove the `===`, which is generally something I do not recommend when working in a typescript context. I elaborated more on this [in another PR review](https://github.com/zetkin/app.zetkin.org/pull/2426#discussion_r1887956255).

Reply:
Thanks a bunch for the feedback. Read through your explanation on the other linked PR aswell!